### PR TITLE
Clarify some things about running from source

### PR DESCRIPTION
**Pull request by Blake-**
This page is hard to follow as is. I tried to clean it up a bit, and add in two important missing bits. One was the way to run everything at once using supervisord. The other was how to run the DB migrations when running things this way.  It's a bit longer, but I think it makes knowing how to do things more clear.

### CODE REVIEW

**Comment by lmcglohon** [comment]
@Blake- Does this change mean we don't need this anymore?
        tech-docs/development/supervisord.md

**Comment by Blake-** [comment]
@lmcglohon Oh yeah, sorry, I meant to mention that in my comment. Maybe dump it? I guess it's not a bad idea to leave that there, just in case. It makes sense to have all that information on the page I added it to here, it might make sense to leave it as a separate page as well? It might also make sense to get rid of that page entirely since it's more or less duplicate content now.

**Comment by trevorthornton** [comment]
@Blake- The additions look good, but I think it would be clearer if the 2 options (starting the component apps individually vs. starting them all with supervisord) were under different sub-headings since they're mutually exclusive. So something like:

## Running a development environment

### Running components individually

(all the instructions before "Use Supervisord...")

### Running all components at once

(then everything you added)

**Comment by Blake-** [comment]
@trevorthornton Ah, yep, makes sense!

**Comment by trevorthornton** [comment]
@Blake- Also, the last part ("You can also clear your database...") might be a third section

**Comment by Blake-** [comment]
@trevorthornton Thanks for those, I've cleaned it up a bit.
